third_party/clang/iOS.cmake

```
@@ -104,6 +108,9 @@ IF (NOT DEFINED IOS_MIN_TARGET)
 ENDIF()
 
 SET(IOS_COMMON_FLAGS "-m${PLATFORM_FLAG_NAME}-version-min=${IOS_MIN_TARGET}")
+if (CATALYST)
+  SET(IOS_COMMON_FLAGS "${IOS_COMMON_FLAGS} -target x86_64-apple-ios13.0-macabi")
```

What if I wish to build for Catalyst with version 14.0 (Big Sur) as minimum supported version? Shouldn't it be like this?
```suggestion
  SET(IOS_COMMON_FLAGS "${IOS_COMMON_FLAGS} -target x86_64-apple-ios${IOS_MIN_TARGET}-macabi")
```

---

Have you found any documentation on `x86_64-apple-ios14.0-macabi`? Does it even exist?

---

You are right, see 27573cb3b5629bb1989a786612ee500f84984b79